isb_lib/models/thing.py

```diff
@@ -143,10 +150,3 @@ class ThingIdentifier(SQLModel, table=True):
     thing_id: Optional[int] = Field(
         default=None, nullable=False, foreign_key="thing._id", index=False
     )
```

Comment:
The ThingIdentifier model class is still here as the migration script uses it to move the data from the identifiers table to the new column.